src/stories/Filter/DurationFilter/DurationFilterDocs.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
```

Comment:
Please also add filter to the AllFilters stories:
1. https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-allfilters--no-filter-bar
2. https://toyotaresearchinstitute.github.io/lakefront/?path=/docs/lakefront-filter-allfilters--with-filter-bar

I believe this is done in the `filterPageUtil.tsx` file.